File: src/wsTripXML/Code/OTAClasses/PAY_ListVCC_RS.vb

'''<remarks/>
<System.Xml.Serialization.XmlTypeAttribute(AnonymousType:=True),
 System.Xml.Serialization.XmlRootAttribute([Namespace]:="", IsNullable:=False)>
Partial Public Class SessionCreateRS
    '''<remarks/>
    Public Property Success As Object

    '''<remarks/>
    Public Property ConversationID As ConversationID

    '''<remarks/>
    <System.Xml.Serialization.XmlAttributeAttribute()>
    Public Property Version As Decimal
End Class

'''<remarks/>
<System.Xml.Serialization.XmlTypeAttribute(AnonymousType:=True)>
Partial Public Class ConversationID
    '''<remarks/>
    Public Property Errors As Errors
End Class

'''<remarks/>
<System.Xml.Serialization.XmlTypeAttribute(AnonymousType:=True)>
Partial Public Class Errors
    '''<remarks/>
    Public Property [Error] As String
End Class

File: src/wsTripXML/Code/OTAClasses/PAY_ListVCC_RS.cs

namespace wsTripXML
{

    /// <remarks/>
    [System.Xml.Serialization.XmlType(AnonymousType = true)]
    [System.Xml.Serialization.XmlRoot(Namespace = "", IsNullable = false)]
    public partial class SessionCreateRS
    {
        /// <remarks/>
        public object Success { get; set; }

        /// <remarks/>
        public ConversationID ConversationID { get; set; }

        /// <remarks/>
        [System.Xml.Serialization.XmlAttribute()]
        public decimal Version { get; set; }
    }

    /// <remarks/>
    [System.Xml.Serialization.XmlType(AnonymousType = true)]
    public partial class ConversationID
    {
        /// <remarks/>
        public Errors Errors { get; set; }
    }

    /// <remarks/>
    [System.Xml.Serialization.XmlType(AnonymousType = true)]
    public partial class Errors
    {
        /// <remarks/>
        public string Error { get; set; }
    }
}